maptivist/src/main/webapp/index.html

```diff
@@ -90,8 +90,9 @@ <h1>Fill in Marker</h1>
                 <br>
                 <input type="text" placeholder="Enter link here" id="marker-link" name="marker-link">
                 <br>
-
-                <button type="button" class="btn" onclick="createMarker()">Submit</button>
```

Comment:
Maybe just turn this into type="submit" so the createMarker() callback gets called to draw the marker on the map?

I think you will want to coordinate with @TBarrett553 for this form.

The issue is that the form submit sends a redirect, which causes the map to reload and the markers are no longer visible on the map. But if the createMarker() uses the fetch API to send the POST in the background, that won't cause a redirect.

Reply:
Yeah for right now, the button works and calls the marker. However, when the form tries to submit there is a null pointer error in the master. But if you tested submitting the form with the improved marker servlet in this PR, I think the problem will be fixed.